projects/cdk/observables/pressed-observable.ts

```diff
@@ -3,15 +3,22 @@ import {filter, mapTo, startWith, switchMapTo, take} from 'rxjs/operators';
 import {mouseDragFinishFrom} from './mouse-drag-finish-from';
 import {typedFromEvent} from './typed-from-event';
 
-export function pressedObservable(element: Element): Observable<boolean> {
+export interface TuiPressedObservableOptions {
+    onlyTrusted: boolean;
+}
+
+export function pressedObservable(
+    element: Element,
+    {onlyTrusted}: TuiPressedObservableOptions = {onlyTrusted: true},
+): Observable<boolean> {
     const {ownerDocument} = element;
 
     if (!ownerDocument) {
         throw new Error('element does not have ownerDocument');
     }
 
     return typedFromEvent(element, 'mousedown').pipe(
-        filter(({isTrusted}) => isTrusted),
+        filter(({isTrusted}) => (onlyTrusted ? isTrusted : true)),
```

Comment:
!onlyTrusted || isTrusted

Reply:
But negative conditions are harder to read in code `G29: Avoid Negative Conditionals` :(
https://eslint.org/docs/rules/no-negated-condition

![image](https://user-images.githubusercontent.com/12021443/134763920-f431ad99-5582-47b3-8755-a8184af3985a.png)

What do you think about?

```ts
    // ...

    return typedFromEvent(element, 'mousedown').pipe(
        filter(({ isTrusted: takeOnlyTrusted }) => {
             const takeAnyEvents: boolean = !onlyTrusted;
             return takeAnyEvents || takeOnlyTrusted;
        }),
```

Reply:
Nope, negatives are just fine. It's insignificant cognitive drain comparing to what we deal here on the daily basis.

Reply:
No problem 😊